src/net/client/mod.rs

```diff
@@ -127,6 +142,25 @@
 //! # }
 //! ```
 
+//! # Limitations
```

Comment:
Do we want to create issues for these and then link to them here?

Reply:
Would that be good if we don't have time to address those issues this year?

Reply:
I don’t know. I mostly want to avoid that we forget about implementing this. But maybe that’ll happen all by itself anyway.